src/main/resources/templates/start.html

```diff
@@ -22,34 +22,13 @@
 }
 </style>
 <body>
-<div id="one">
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <h2> Meal Mentor </h2>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav mr-auto">
-            <li class="nav-item active">
-                <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
-            </li>
-            <li class="nav-item active">
-                <a class="nav-link" href="#">Shopping List</a>
-            </li>
-        </ul>
-        <form class="form-inline my-2 my-lg-0" action ="#"  th:action="@{/searchReceipe}" th:object="${Recipe}" method="GET">
-            <input class="form-control mr-sm-2" type="search" placeholder="Search a meal ..." aria-label="Search" th:field="*{name}" >
-            <button class="btn btn-outline-info my-2 my-sm-0" type="submit">Search</button>
-        </form>
-    </div>
-</nav>
+<div th:replace="fragments/topnav :: navbar">
 </div>
-<div id="two">
+<div>
     <table class="table table-sm">
         <thead>
         <tr style="LINE-HEIGHT:60px">
-            <th scope="col"></th>
+            <th scope="col"><a href="#"> <select id="sel"> <option value=""> -- Select -- </option> </select> </a></th>
```

Comment:
This might be wrong to move but since it is an unimplemented feature I am not sure what it does. I moved it to a spot that looks better but I would overall recommend not having it in the main branch until you are ready to use it.